src/pgrn-row-level-security.c

```diff
@@ -101,6 +101,10 @@ PGrnCheckRLSEnabledSeqScan(FunctionCallInfo fcinfo)
 	if (!portal) {
 		return false;
 	}
+	if (!(portal->queryDesc)) {/* For EXPLAIN ANALYZE */
+		/* If portal->queryDesc is NULL, PostgreSQL's Executor is not active. */
+		return false;
```

Comment:
Can we return `false` here?
PGroonga functions may show arguments in error message only when `EXPLAIN ANALYZE`?

Reply:
I have already returned false here.

Reply:
I wanted to say that `false` may not be good here.
``EXPLAIN ANALYZE SELECT * FROM table WHERE column &` 'xxx'`` will show the given argument in error message with `false`.